package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+        "start": "webpack serve --open 'google-chrome'",
```

Comment:
- [ ] Kindly ensure you setup your start command correctly , am getting this error when I run it,

![image](https://user-images.githubusercontent.com/61408860/183952050-91e2821c-76ab-42d8-b9d9-0371742335ae.png)

Reply:
@danmainah can you check it now? I corrected the google-chrome section.